Core/HLE/sceKernelModule.cpp

```diff
@@ -1280,8 +1280,9 @@ static Module *__KernelLoadELFFromPtr(const u8 *ptr, size_t elfSize, u32 loadAdd
 			if (end > module->textEnd)
 				module->textEnd = end;
 
-			if (scan)
-				MIPSAnalyst::ScanForFunctions(start, end, !gotSymbols);
+			if (scan) {
+				MIPSAnalyst::ScanForFunctions(start, end, insertSymbols);
```

Comment:
When reading this code, it's super unclear that ScanForFunctions actually can modify insertSymbols. Making it a pointer argument instead of a reference would make that clear .. though it's still an I/O argument, which I don't love but can live with :)